vendor/plugins/edu.harvard.dce.paella.captionsPlugin/captionsPlugin.js

```diff
@@ -44,6 +46,19 @@ Class ("paella.plugins.DceCaptionsPlugin", paella.ButtonPlugin,{
 	setup:function() {
 		var self = this;
 
+		self._activeCaptions = paella.captions.getActiveCaptions();
+
+		self._searchOnCaptions = self.config.searchOnCaptions || false;
+
+		// MATT-2219 #DCE Assume no caption text if first language has no caption text
+		var id = paella.captions.getAvailableLangs()[0].id;
```

Comment:
so there will always be something in `getAvailableLangs()` even for videos that don't have generated captions?